app/routers/profile.py

```diff
@@ -46,6 +60,12 @@ async def profile(
 
     session.close()
```

Comment:
Are you sure you need to close the session?

Reply:
fixed

Reply:
I still see it in the code. Forgot to push, maybe?